TombEngine/Scripting/Internal/ReservedScriptNames.h

```diff
@@ -318,6 +318,7 @@ static constexpr char ScriptReserved_EmitLightningArc[]				= "EmitLightningArc";
 static constexpr char ScriptReserved_EmitShockwave[]				= "EmitShockwave";
 static constexpr char ScriptReserved_EmitLight[]					= "EmitLight";
 static constexpr char ScriptReserved_EmitSpotLight[]				= "EmitSpotLight";
+static constexpr char ScriptReserved_EmitFxFog[]					= "EmitFxFog";
```

Comment:
In my opinion, we should stick to original TRLE terminology here. "EmitFxFog" sounds like something that TRLE users are not familiar with. I suggest renaming to "EmitFogBulb"